configure.py

```diff
@@ -565,6 +565,7 @@ def find_ninja():
     'test/boost/token_metadata_test',
     'test/boost/top_k_test',
```

Comment:
> and before releases. Unfortunately we don't have a precedent for
something like that.

@mykaul let's start work on an intermediate category of tests like this one. We already have tests that benefit from longer runtime but don't merit extending CI time for them.